.github/copilot-instructions.md

```diff
@@ -0,0 +1,157 @@
+
+### Build and Development
+
+- Build website: `cd website && npm run build` -- takes 1.4 seconds. Set timeout to 1+ minute.
+- Run website dev server: `cd website && npm run dev` -- starts immediately, runs on localhost:4321
+- Preview built website: `cd website && npm run preview`
+
+## Main Functionality
+
+### Event Management Scripts
+
+Run from root directory:
+
+- **Create calendar entries**: `npm run create-calendar-entries <PATH_TO_CSV>`
+  - Reads CSV files from `/input-csvs/` directory
+  - Creates Google Calendar events for church planning
+  - Requires Google API credentials in .env
+- **Update website slider**: `npm run update-slider`
+  - Automates WordPress admin login and content updates
+  - Uses CodeceptJS with Playwright for browser automation
+  - Requires WordPress credentials and WP_SLIDER_URL in .env
+  - Runs headless by default (HEADLESS=true)
+
+- **Create YouTube broadcasts**: `npm run create-youtube-broadcasts <PATH_TO_CSV>`
+  - Creates YouTube live broadcasts from CSV planning data
+  - Requires YouTube API credentials
+- **Get next Sunday date**: `npm run get-next-sunday`
+  - Utility script for date calculation
+  - Used by other automation scripts
+
+### CodeceptJS Testing
+
+- Run website automation tests: `npm run codeceptjs`
+- Run headless tests: `npm run codeceptjs:headless`
+- Configuration: `src/website_update/codecept.conf.js`
+- Tests target WordPress admin interface at lkg-halle.de
+
+## Validation
+
+### Before Making Changes
+
+- **ALWAYS run formatting check**: `npx prettier --check .`
+- **ALWAYS run formatting fix before committing**: `npx prettier --write .`
+- **Test website build**: `cd website && npm run build`
+- **Test website dev server**: `cd website && npm run dev`
+
+### After Making Changes
+
+- **Manual validation scenarios**:
+  1. **Website functionality**: Start dev server, verify pages load at localhost:4321
+  2. **Build process**: Ensure `npm run build` completes without errors
+  3. **Script syntax**: Test `npm run get-next-sunday` (will show auth error but proves syntax is correct)
+  4. **Formatting**: Run `npx prettier --check .` and fix any issues
+
+### CI/CD Integration
+
+- GitHub Actions automatically run on schedule (Sundays 18:00 UTC)
+- Actions require all secrets configured in repository settings
+- Dependabot updates dependencies monthly with grouped minor/patch updates
+
+## Project Structure
+
+### Root Directory Scripts
+
+- `/src/calendar_update/` - Google Calendar API integration
+- `/src/website_update/` - WordPress automation with CodeceptJS
+- `/src/youtube_update/` - YouTube API integration
+- `/src/csv-parser/` - CSV file processing utilities
+- `/input-csvs/` - Event planning CSV files
+
+### Website Directory
+
+- `/website/src/pages/` - Astro page components
+- `/website/src/layouts/` - Astro layout components
+- `/website/public/` - Static assets
+- `/website/dist/` - Built website output (generated)
+
+## Common Development Tasks
```

Comment:
What's this section needed for? Just listing the files in the root directory doesn't feel super useful. And, also the dependencies are listed in package.json. You told me, you already understand that this is a node-js project (also, as stated in the preamble). So duplicating this here doesn't feel super useful to me